package.json

```diff
@@ -63,9 +63,6 @@
     "url": "https://github.com/gpbl/react-day-picker/issues"
   },
   "homepage": "https://react-day-picker.js.org",
-  "peerDependencies": {
-    "react": "~0.13.x || ~0.14.x || ^15.0.0"
```

Comment:
Mentioned in the PR description, but I'll copy/paste here:
> I also had issues testing changes to the library. When I ran npm link to test my changes, I kept getting errors saying that webpack could not find the "react" library. I looked and saw that React is listed as a peerDependency. However, since React is imported by the library's code and is required in order for this library to compile properly, I'm thinking it should be a dependency.